src/and-cli-restore-azure-storage.ts

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env node
+
+import { CommandRunner } from "./modules/command-runner";
+import { Echo } from "./modules/echo";
+import program from "commander";
+import shell from "shelljs";
+import { AzureAzcopySync } from "./modules/azure-azcopy-sync";
+
+CommandRunner.run(async () => {
+    // -----------------------------------------------------------------------------------------
+    // #region Variables
+    // -----------------------------------------------------------------------------------------
+
+    let deleteDestination: boolean = false;
+    let destinationAccount: string;
+    let destinationContainer: string;
+    let destinationSasToken: string;
+    let recursive: boolean = false;
+    let sourceAccount: string;
+    let sourceContainer: string;
+    let sourceSasToken: string;
+
+    // #endregion Variables
+
+    /// -----------------------------------------------------------------------------------------
+    // #region Functions
+    // -----------------------------------------------------------------------------------------
+
+    const restoreAzureStorage = {
+        description() {
+            return "Restore application assets in Azure Storage";
+        },
+        run() {
+            // Check system/command requirements
+            this.validateOrExit();
+
+            /*
+                Following options currently only support syncing utilizing Azure SAS Tokens.
+                Consider future refactor to allow this restore aspect to first generate a SAS
```

Comment:
Consider adding an issue to the repo and referencing it here.

Reply:
Good call.  Added https://github.com/AndcultureCode/AndcultureCode.Cli/issues/150